modulefiles/module_base.wcoss2

```diff
@@ -0,0 +1,42 @@
+#%Module######################################################################
+##
+
+module load envvar/1.0
+module load PrgEnv-intel/8.1.0
+module load craype/2.7.8
+module load intel/19.1.3.304
+module load cray-mpich/8.1.7
+module load cray-pals
```

Comment:
curious, what is `cray-pals`?

Reply:
Per Carolyn's presentation slide on "Running jobs":

```
• PBS Pro uses the Cray Parallel Application Launch Service (PALS)
to launch MPI jobs
• cray-pals is a module that needs to be loaded to access mpiexec
```

I haven't gotten to testing job submission yet, have added the module in anticipation of needing it at runtime.

```
module show cray-pals
----------------------------------------------------------------------------------------------
   /opt/cray/pe/lmod/modulefiles/core/cray-pals/1.0.12.lua:
----------------------------------------------------------------------------------------------
whatis("PALS - Parallel Application Launch Service")
prepend_path("PATH","/opt/cray/pe/pals/1.0.12/bin")
prepend_path("MANPATH","/opt/cray/pe/pals/1.0.12/man")
set_alias("mpirun","mpiexec")
```